source/api_documentation/network_integration/advertiser_campaigns/_get_advertiser_campaign_unarchive.rst

```diff
@@ -0,0 +1,15 @@
+.. container:: endpoint-long-description
```

Comment:
@mikeweaver 
I discussed with @victorborda if it would be possible to rip out other non-RESTful API endpoints from the platform (and from the documentation).
 
There are a couple problems:
1. Customers might be using those as they stand today. If we deprecate them in the API we'll likely have a customer facing issue. 
2. Some customers may have old platforms and may not be able to use the correct HTTP verb, so we provide broader support while unfortunately breaking REST conventions. 

Given that, do we want to 
A) Remove the documentation but not the functionality? 
or 
B) Keep the documentation as it is and add this "unarchive" example?